spec/lcm/helpers/app_store_helper.rb

```diff
@@ -0,0 +1,106 @@
+require 'gooddata'
+
+module GoodData
+  module AppStore
+    module Helper
+      def self.download_file_as_csv(project, filename)
+        file = StringIO.new
+        project.download_file(filename, file)
+        CSV.parse(file.string)
+      end
+
+      def self.upload_file(project, data)
+        begin
+          tempfile = Tempfile.new('file_upload')
+          path = tempfile.path
+
+          CSV.open(path, 'w') do |csv|
+            data.each { |row| csv << row}
+          end
+          project.upload_file(path)
+        ensure
+          tempfile.unlink
+        end
+        path
+      end
+
+      def self.remove_segment(domain, segment_name)
+        begin
+          domain.segments(segment_name).delete(force: true)
+        rescue
+          # Segment already deleted
+        end
+      end
+
+      def self.remove_test_projects(client, slug)
+        old_projects = client.projects.select do |project|
+          project.title.include?(slug)
+        end
+        old_projects.peach(&:delete)
+      end
+
+      def self.create_simple_project(title, client, token)
+        blueprint = GoodData::Model::ProjectBlueprint.build(title) do |p|
+          p.add_date_dimension('committed_on')
+          p.add_dataset('devs') do |d|
+            d.add_anchor('attr.dev')
+            d.add_label('label.dev_id', :reference => 'attr.dev')
+            d.add_label('label.dev_email', :reference => 'attr.dev')
+          end
+          p.add_dataset('commits') do |d|
+            d.add_anchor('attr.commits_id')
+            d.add_fact('fact.lines_changed')
+            d.add_date('committed_on')
+            d.add_reference('devs')
+          end
+        end
+        project = GoodData::Project.create_from_blueprint(blueprint, auth_token: token, client: client)
+
+        # Load data
+        commits_data = [
```

Comment:
Similar code found in app_store_helper.rb:60, full_project_spec.rb:133 (mass = 36)